example/index.html

```diff
@@ -72,8 +82,7 @@
   {name:"Watch a movie", date:	"2012-09-28T20:16:00"},
   {name:"Go to sleep", date:	"2012-09-28T23:00:00"}
 ];
-TimeKnots.draw("#timeline2", mySchedule, {horizontalLayout: false, color: "#669", height: 450, width:200, showLabels: true, labelFormat:"%H:%M"});
-
+TimeKnots.draw("#timeline2", mySchedule, {horizontalLayout: false, colorTimeline: "#669", height: 450, width:200, showLabels: true, labelFormat:"%H:%M"});
```

Comment:
I'm also curious why this timeknow seem to move the tip to (0,0) when dissapearing?